docs/ext/files.rst

```diff
@@ -0,0 +1,48 @@
+.. _ext-files:
+
+************
+Mopidy-Files
+************
+
+Mopidy-Files is an extension for playing music from your local music archive.
+It is bundled with Mopidy and enabled by default. 
+It allows you to browse through your local file system.
+Only files that are considered playable will be shown
+
+This backend handles URIs starting with ``file:``.
+
+
+Configuration
+=============
+
+See :ref:`config` for general help on configuring Mopidy.
+
+.. literalinclude:: ../../mopidy/files/ext.conf
+    :language: ini
+
+.. confval:: files/enabled
+
+    If the files extension should be enabled or not.
+
+.. confval:: files/media_dir
+
+    A list of directories to be browsable.
+    Each Directory path has to be written in a separate line.
```

Comment:
This isn't strictly true, as our config parser supports both:

   media_dir = foo, bar, baz

And:

    media_dir =
        foo
        bar
        baz

Also, "Directory" shouldn't be capitalized.